src/app/b-e08a-event-home/e08a-event-home.component.html

```diff
@@ -1,2 +1,41 @@
-<p>e08a-event-home works!</p>
-<img src="/src/assets/images/design-photos/c05c-event-landing.png" alt="'Event Landing Image" />
+<p class="uppercase text-4xl font-medium text-center mt-12">
+    event name
```

Comment:
@theJayBecker 

do you feel up to, adding values in these places?

Like in the 

```
export class E08aEventHomeComponent {
  eventName = 'Dragoncon Electronic Frontiers Forums track';
}
```
Then in line 2, here where it says 'event name' you would add,

`{{eventName}}`